src/components/CaseStudies/images.tsx

```diff
@@ -62,7 +62,12 @@ export const manomanoLogoImage = (
 )
 
 export const rakutenHeroImage = (
-  <StaticImage src="../../assets/case-studies/rakuten-team.jpg" alt="Femtasy banner" width={680} />
+  <StaticImage
+    src="../../assets/case-studies/rakuten-team.jpg"
+    alt="Femtasy banner"
```

Comment:
Let's fix the `alt` text here, it should be "Rakuten team"

Reply:
adjusted, and for the `lokalportal` too